src/GafferScene/Instancer.cpp

```diff
@@ -325,10 +325,10 @@ struct IdData
 // with a grouping pattern that can be changed with seedScramble
 int seedForPoint( int index, const IdData &idData, int numSeeds, int seedScramble )
```

Comment:
Should `index` be `size_t` here?

Reply:
That index is a physical index, not an id. I'm pretty sure we have lots of other places where we don't allow PointsPrimitives to have more than 2^31 points, so this is probably fine?

Reply:
Physical indices in PointsPrimitives are `size_t`, both because PointsPrimitive::m_numPoints` is `size_t` and also because primitive variable (`std::vector`) size is `size_t`.